src/routes/(protected)/api/tmdb/search/[type]/+server.ts

```
@@ -0,0 +1,286 @@
+				queryParams[key] = value;
+			}
+		}
+
+		// Route to appropriate endpoint based on searchMode
+		if (searchMode === "search" || searchMode === "hybrid") {
+			console.log("Search with the following params", queryParams)
+			// Use search endpoint
+			if (type === "movie") {
+				const searchResult = await providers.tmdb.GET("/3/search/movie", {
+					params: {
+						query: queryParams as any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)